source/controlTypes/state.py

```diff
@@ -204,6 +205,9 @@ def negativeDisplayString(self) -> str:
 	State.HASPOPUP_LIST: _("opens list"),
 	# Translators: Presented when a control has a pop-up tree.
 	State.HASPOPUP_TREE: _("opens tree"),
+	# Translators: Presented when a link destination points to the page containing the link.
+	# For example, links of a table of contents of a document with different sections.
+	State.INTERNAL_LINK: _("same page"),
```

Comment:
Could you also add an entry to the braille labels for state? I think `sp` or `smp` will suffice